### PR TITLE
consolidation

### PR DESCRIPTION
**Pull request by emanuelshalev**
code consolidation
text to speech update
requirements update

### CODE REVIEW

File: companion_app/main.py [hunk 1]
```diff
@@ -8,12 +8,11 @@
...
 import pyautogui
 from gtts import gTTS
-from mpg123 import Mpg123, Out123
-
+import pyttsx3
 
 def get_screen_image():
     with NamedTemporaryFile() as f:
-        pil_image = pyautogui.screenshot(imageFilename=f.name)
+        pil_image = pyautogui.screenshot(imageFilename=f.name+'.png')
```
Comment (keyan) [inline companion_app/main.py:15]: Why did you add `.png` here? The passing of a filename is just so a temporary file is used and then deleted.

**Comment by keyan** [comment]
How much does this improve the TTS time?

File: companion_app/main.py (hunk 1 above)
Comment (emanuelshalev) [reply companion_app/main.py:15]: I think its Windows related. It was crashing trying to create a file without extension.

**Review by keyan — COMMENTED** [review]
looks good
